src/main/proto/opencdc.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
```

Comment:
Suggestion: using `buf` could make it easier to manage the proto changes. The java options can be injected in [managed mode](https://docs.buf.build/generate/managed-mode#java). The benefit is you wouldn't have to copy over the proto files in the repo as they can be fetched by buf on demand from the schema registry (https://buf.build/conduitio/conduit-connector-protocol) and updating to a newer version will be easier.

This is out of scope of this PR, it's just a thing I noticed.